src/cmd/build.js

```diff
@@ -175,7 +179,6 @@ export type BuildCmdParams = {|
   sourceDir: string,
   artifactsDir: string,
   asNeeded?: boolean,
-  noInput?: boolean,
```

Comment:
I think it would be unfortunate to remove this from the type. It's true to say that it will be a possible param since that's how yargs will send it. I wonder if we could solve it instead by defining all global options in one place and referencing it? Example:

```js
import type {GlobalOptions} from '../somewhere';

export type BuildCmdParams = {|
  asNeeded?: boolean,
  overwriteDest?: boolean,
  ignoreFiles?: Array<string>,
  ...GlobalOptions,
|};
```